assets/locales/service.en.toml

```diff
@@ -125,6 +125,26 @@ one = "Bulletin"
 description = "Released"
 one = "Released"
 
-[StatusLinePreviousReleases]
+[StatusLineViewPreviousReleases]
 description = "View previous releases"
-one = "View previous releases"
+other = "View previous releases"
+
+[StatusLineViewCorrectedVersion]
+description = "View corrected version"
+one = "View corrected version"
+
+[StatusLineViewLatestRelease]
+description = "View latest release"
+one = "View latest release"
+
+[StatusLineReleaseLatest]
+description = "LATEST"
+one = "LATEST"
```

Comment:
Using uppercase [might be difficult](https://a11y-101.com/design/capitalisation) for users with accessibility needs. Could you keep the text as lower/title case and then use css (`ons-u-tt-u`) to transform?